symengine/sets.h

```diff
@@ -34,7 +31,7 @@ class EmptySet : public Set {
     IMPLEMENT_TYPEID(EMPTYSET)
     //EmptySet(EmptySet const&) = delete;
     void operator=(EmptySet const&) = delete;
-    static RCP<const EmptySet> getInstance();
```

Comment:
Yes, but the method name is `getInstance` which is supposed to return an instance of `Emptyset`, but it returns a `RCP<const Set>` instead.
Also, if you want an `RCP<const EmptySet>`, you can't get it.